.eslintrc.yml

```diff
@@ -1,20 +1,12 @@
----
-env:
-  jest/globals: true
 plugins:
   - flowtype
-  - jest
 extends:
-  - standard
-  - plugin:import/warnings
-  - plugin:import/errors
+  - "@enkidevs/eslint-config-backend"
   - plugin:flowtype/recommended
 parser: babel-eslint
 rules:
-  jest/no-disabled-tests: 1
-  jest/no-focused-test: 0
-  jest/no-identical-title: 2
-  jest/valid-expect: 2
+  node/no-unsupported-features/es-syntax: 0
```

Comment:
You can fix this one by adding to `package.json`:

```json
  "engines": {
    "node": "8.10.0"
  },
```

Reply:
magic!

Reply:
I added it but eslint seemed to ignore it, so i left the rule

Reply:
Imo this should be ok, since they should be mutually inclusive